components/AudioSettings/MuteButton.js

```diff
@@ -34,34 +34,27 @@ const StyledContainer = styled.div`
   position: fixed;
```

Comment:
I think it would be more organized if you would put the styles in a separate file, like in your other components

Reply:
Mathis is working on it